clip_obs        : 100
clip_actions    : 100
damping         : 1.
stiffness       : 40.0
action_scale    : 0.25
model_path      : "a1_flat/model_500.pt"
num_observations: 48

lin_vel: 2.0
ang_vel: 0.25
dof_pos: 1.0 
dof_vel: 0.05
imu: 0.1
height_measurements: 5.0
friction_measurements: 1.0
body_height_cmd: 2.0
gait_phase_cmd: 1.0
gait_freq_cmd: 1.0
footswing_height_cmd: 0.15
body_pitch_cmd: 0.3
body_roll_cmd: 0.3
aux_reward_cmd: 1.0
compliance_cmd: 1.0
stance_width_cmd: 1.0
stance_length_cmd: 1.0
segmentation_image: 1.0
rgb_image: 1.0
depth_image: 1.0

              # hip, thigh, calf
torque_limits: [20., 55., 55.,  # front left
                20., 55., 55.,  # front right
                20., 55., 55.,  # rear  left
                20., 55., 55. ] # rear  right

                    
default_dof_pos: [ 
                # hip,   thigh,  calf
                 0.1000,  0.8000, -1.5000, # front left
                -0.1000,  0.8000, -1.5000, # front right
                 0.1000,  1.0000, -1.5000, # rear  left
                -0.1000,  1.0000, -1.5000  # rear  right
                ]

joint_names  : [
        "FL_hip_joint"  ,
        "FL_thigh_joint",
        "FL_calf_joint" ,
        "FR_hip_joint"  ,
        "FR_thigh_joint",
        "FR_calf_joint" ,
        "RL_hip_joint"  ,
        "RL_thigh_joint",
        "RL_calf_joint" ,
        "RR_hip_joint"  ,
        "RR_thigh_joint",
        "RR_calf_joint" ,
        ]

algorithm:
        clip_param            : 0.2
        desired_kl            : 0.01
        entropy_coef          : 0.01
        gamma                 : 0.99
        lam                   : 0.95
        learning_rate         : 0.001
        max_grad_norm         : 1.0
        num_learning_epochs   : 5
        num_mini_batches      : 4
        schedule              : 'adaptive'
        use_clipped_value_loss: True
        value_loss_coef       : 1.0            

init_member_classes: {}

policy:
        activation        : 'elu'
        actor_hidden_dims : [128, 64, 32]
        critic_hidden_dims: [128, 64, 32]
        init_noise_std    : 1.0            

runner: 
        algorithm_class_name: 'PPO'
        checkpoint          : -1
        experiment_name     : 'go1_rough'
        load_run            : -1
        max_iterations      : 10000
        num_steps_per_env   : 24
        policy_class_name   : 'ActorCritic' 
        resume              : True 
        resume_path         : None
        run_name            : 'go1_rough'
        save_interval       : 50
                
        


